src/OrchardCore.Modules/OrchardCore.Deployment/Views/Import/Json.cshtml

```diff
@@ -0,0 +1,27 @@
+<h1>@RenderTitleSegments(T["Import JSON"])</h1>
```

Comment:
Suggest JSON Import to keep the translations the same as the menu item?

Reply:
ok